### PR TITLE
.br updated - 18 new 2nd level city domains

### PR DESCRIPTION
**Pull request by fnevesbr**

### CODE REVIEW

**Comment by daftano** [comment]
@fnevesbr I opened a PR onto your fork for rio.br: did you forgot it, or it is on purpose according to same decision of the registry?

**Comment by fnevesbr** [comment]
@daftano, it is on purpouse. We'll include it when we've a firm deployment date.

**Comment by daftano** [comment]
thank you 👍

File: public_suffix_list.dat [hunk 1]
```diff
@@ -472,26 +479,36 @@ jor.br
...
 odo.br
 org.br
+osasco.br
+palmas.br
 poa.br
 ppg.br
 pro.br
 psc.br
 psi.br
+pvh.br
```
Comment (sleevi) [inline public_suffix_list.dat]: I cannot find supporting documentation for this on the https://registro.br/dominio/categoria.html page

**Comment by sleevi** [comment]
Waiting followup regarding pvh.br

File: public_suffix_list.dat (hunk 1 above)
Comment (fnevesbr) [inline public_suffix_list.dat]: pvh.br was withdraw last friday. I'll update the PR.

**Comment by fnevesbr** [comment]
.pvh.br was withdrawn last friday. Will be include in a future update.

**Comment by rhulio** [comment]
Please add: caxias.br and the.br thank you

**Pull request merged**